bin/base.sh

```diff
@@ -50,6 +50,17 @@ else
     }
 fi
 
+# Use fuse low-level API if it's available. ("command -v" is POSIX.1-2008.)
+if ( command -v squashfuse_ll >/dev/null 2>&1 ); then
+    squashfuse_ () {
+        squashfuse_ll "$@"
+    }
+else
+    squashfuse_ () {
+        squashfuse "$@"
+    }
+fi
+
```

Comment:
Do we handle the case where there is no squash installed at all?

Reply:
If no squashfuse is found an error will be produced as is
`$ ch-mount spark.sqfs test/
/usr/local/bin/ch-mount: 60: /usr/local/bin/ch-mount: squashfuse: not found`